interface/client/templates/popupWindows/onboardingScreen.js

```diff
@@ -322,6 +322,18 @@ The onboardingScreen password template
 @constructor
 */
 
+Template['popupWindows_onboardingScreen_password'].helpers({
+    /**
+    Show password
+
+    @method showPassword
+    */
+    'showPassword': function() {
```

Comment:
I copied this helper function from `popupWindows_onboardingScreen_importAccount` helpers.
Would you change that one too or do I miss your argument?